src/ontology/mondo-ingest.Makefile

```diff
@@ -476,6 +488,7 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_map
 	--mondo-terms-path $(REPORTDIR)/mirror_signature-mondo.tsv \
 	--slurp-dir-path slurp/ \
 	--outpath $@
+.PRECIOUS: slurp/%.tsv
```

Comment:
## More **`rm` issue**s
- [x] `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv slurp/icd11foundation.tsv`
- [ ] `imports/ro_terms_combined.txt` https://github.com/monarch-initiative/mondo-ingest/issues/477

I checked the build log from last week, and at the end it showed `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv slurp/icd11foundation.tsv imports/ro_terms_combined.txt`. After adding that 1 `.PRECIOUS`, now the slurp file is no longer being removed, but the other two still are: `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv imports/ro_terms_combined.txt`. I think we want these, so I'm going to apply the same fix for both of those as well. I pasted more of the log from `build-mondo-ingest` below just in case it's of any value.

<details><summary>Log related to (3)</summary>
<p>

I have the full logs saved as `.txt` files also if interested

From the build last week:
```
Release files are now in ../.. - now you should commit, push and make a release         on your git hosting site such as GitHub or GitLab
rm reports/icd11foundation_exclusion_reasons.robot.template.tsv slurp/icd11foundation.tsv imports/ro_terms_combined.txt
make[1]: Leaving directory '/work/src/ontology'
Mondo Ingest has been fully completed
```

From the build today:
```
Release files are now in ../.. - now you should commit, push and make a release         on your git hosting site such as GitHub or GitLab
rm reports/icd11foundation_exclusion_reasons.robot.template.tsv imports/ro_terms_combined.txt
make[1]: Leaving directory '/work/src/ontology'
Mondo Ingest has been fully completed
```

</p>
</details> 

I've followed up on these two remaining issues here:
- Thread: [Bug fix: `rm reports/icd11foundation_exclusion_reasons.robot.template.tsv`](https://github.com/monarch-initiative/mondo-ingest/pull/434#discussion_r1533134445)
- #477